src/Agent/NewRelic/Agent/Core/Errors/ErrorData.cs

```diff
@@ -26,5 +27,12 @@ public ErrorData(string errorMessage, string errorTypeName, string stackTrace, D
             Path = null;
             CustomAttributes = customAttributes;
         }
+
+        public ErrorData(string errorMessage, string errorTypeName, string stackTrace, DateTime noticedAt, ReadOnlyDictionary<string, object> customAttributes, bool isExpected)
```

Comment:
Let's come back to this and go with one constructor. I understand you added this one so as to avoid refactoring all the tests as part of this PR. Can you simplify this and do the refactor of the tests in a follow up PR?

Reply:
Yes, I will combine these constructors in a different PR.